legacy/src/main/java/org/pentaho/di/job/entries/hadooptransjobexecutor/JobEntryHadoopTransJobExecutor.java

```diff
@@ -754,13 +756,10 @@ public Result execute( Result result, int arg1 ) throws KettleException {
       String numMapTasksS = environmentSubstitute( numMapTasks );
       try {
         if ( Integer.parseInt( numMapTasksS ) < 0 ) {
-          throw new KettleException(
-              BaseMessages.getString( PKG, "JobEntryHadoopTransJobExecutor.NumMapTasks.Error" ) );
-        }
-      } catch ( NumberFormatException e ) {
-        if ( log.isDebug() ) {
```

Comment:
@stepanovdg why are we removing the number format logging when loglevel is debug?